pkg/intermediate/types.go

```diff
@@ -33,11 +33,15 @@ type AggregationFlowRecord struct {
 	// inter-node flow and record from the node for the case of intra-node flow.
 	ReadyToSend               bool
 	waitForReadyToSendRetries int
-	// isMetaDataFilled is an indicator for IPFIX Mediator to check whether K8s
+	// areCorrelatedFieldsFilled is an indicator for IPFIX Mediator to check whether K8s
 	// metadata are filled for flow record. It is always true for Intra-Node
 	// and ToExternal flows and only applicable for Inter-Node flows that are
 	// not required to be correlated. (e.g. flows with Egress deny rule applied)
-	isMetaDataFilled bool
+	areCorrelatedFieldsFilled bool
+	// Some fields like Pod labels will be filled on the Flow Aggregator side,
+	// hasFilledExternalFields is an indicator to check whether these fields
+	// has been filled on Flow Aggregator.
```

Comment:
Could you rephrase this comment so that it is applicable for a generic IPFIX intermediate process and not specific to Flow Aggregator which is part of Antrea? 
Please mentioned that setting and utilizing this field is upto the user and not used in go-ipfix library code.